services/QuillLMS/app/controllers/profiles_controller.rb

```diff
@@ -190,6 +191,13 @@ def staff
     @act_sesh_records.first
   end
 
+  protected def completed_evidence_activity_prior_to_scoring
+    ActivitySession
+      .where(user_id: current_user.id, activity_id: Activity.evidence.ids)
+      .where("completed_at < ?", DateTime.new(2024, School::SCHOOL_YEAR_START_MONTH, 1))
+      .present?
```

Comment:
This isn't super-obvious from the Rails code, but this particular approach makes two database calls: one to get a list of Activity IDs, and then a second one to look up ActivitySessions.  Given how often this looks like it might get called, it's probably worth making this slightly more efficient if we can.  An alternative version of this query which removes that first query looks like this:
```
ActivitySession
  .joins(activity: :classification)
  .where(activity: {classification: ActivityClassification.evidence})
  .where(user_id: current_user.id)
  .where("completed_at < ?", DateTime.new(2024, School::SCHOOL_YEAR_START_MONTH, 1))
  .present?
```